packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -0,0 +1,81 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "controls": {
+        "gap": {
+          "value": "var(--base-size-16)"
+        },
+        "background": {
+          "value": "hsl(var(--base-color-scale-black-0-hsl) / 80%)"
+        },
+        "backgroundGradient": {
+          "value": "linear-gradient(rgba(0, 0, 0, 0), hsl(var(--base-color-scale-black-0-hsl) / 40%))"
+        },
+        "padding": {
+          "value": "var(--base-size-16) var(--base-size-24)"
+        },
+        "icon": {
+          "height": {
+            "value": "24px"
+          }
+        },
```

Comment:
I don't think this is being used? Can we remove this please? I don't think the icon size needs to be configurable.